archive-press_quote.php

```diff
@@ -0,0 +1,71 @@
+<?php
+get_header();
+?>
+
+<main id="main-content">
+  <section id="posts">
+    <div class="container">
+
+<?php
+if( have_posts() ) {
+  $i = 0;
+
+  while( have_posts() ) {
+    the_post();
+    $quote = get_post_meta($post->ID, '_igv_quote_text', true);
```

Comment:
id say we should wrap the quote text in `" "` quotes here, so client doesn't add them in meta field